HOST lewis.lbl.gov
only_i 337
only_code None
force_amber_prep True
dry_run False
... 337 8icn
/net/chevy/raid1/nigel/amber/hi_lo
/net/chevy/raid1/nigel/amber/hi_lo/ic/8icn
8icn.pdb 4phenix_8icn.pdb
local_archive None
archive PDB None
Reading from local mirror:
  /net/cci/pdb_mirror/pdb/ic/pdb8icn.ent.gz
Model saved to /net/lewis/raid1/scratch1/nigel/amber/hi_lo/ic/8icn/8icn.pdb
Traceback (most recent call last):
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 199, in <module>
    run2(sys.argv[1:])
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 155, in run2
    files.append(get_pdb(id, data_type_, mirror, log, format=data_format))
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 213, in get_pdb
    data = fetch(id, data_type, mirror=mirror, format=format, log=log)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 150, in fetch
    data = libtbx.utils.urlopen(url)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/libtbx/utils.py", line 2069, in urlopen
    return urllib2.urlopen(*args, **kwds)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 110] Connection timed out>
local_archive None
archive PDB None
Reading from local mirror:
  /net/cci/pdb_mirror/pdb/ic/pdb8icn.ent.gz
Model saved to /net/lewis/raid1/scratch1/nigel/amber/hi_lo/ic/8icn/8icn.pdb
Traceback (most recent call last):
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 199, in <module>
    run2(sys.argv[1:])
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 155, in run2
    files.append(get_pdb(id, data_type_, mirror, log, format=data_format))
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 213, in get_pdb
    data = fetch(id, data_type, mirror=mirror, format=format, log=log)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 150, in fetch
    data = libtbx.utils.urlopen(url)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/libtbx/utils.py", line 2069, in urlopen
    return urllib2.urlopen(*args, **kwds)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 110] Connection timed out>

==================================================
Summary of pdb4amber for: 8icn.pdb
===================================================

----------Chains
The following (original) chains have been found:
A
P
T

---------- Alternate Locations (Original Residues!))

The following residues had alternate locations:
None

---------- Mising heavy atom(s)

None
not prmtop_filename /net/lewis/scratch1/nigel/amber/hi_lo/ic/8icn/4amber_8icn.prmtop
not prmtop_filename /net/chevy/raid1/nigel/amber/hi_lo/ic/8icn/4amber_8icn.prmtop

	Running AmberPrep : 8icn.pdb


==================================================
Running pdb4amber on 8icn.pdb
==================================================

==================================================
Setting up library files for non-standard residues
==================================================

==================================================
Running elbow/antechamber for ATP 
==================================================

MoleculeClass :  C:10  N: 5  O:13  P: 3 (CHEMICAL COMPONENTS format)
	43 atoms
	45 bonds
	78 angles
	104 dihedrals
	2 rings (groups, rings, atoms) 3 16
	4 chirals : R S R R 

~> antechamber -i 4antechamber_ATP.pdb -fi pdb -o ATP.mol2 -fo mol2       -nc -4 -m 1 -s 2 -pf y -c bcc -at gaff2 -ek "qm_theory='AM1',grms_tol=0.0005,scfconv=1.d-10,maxcyc=0,ndiis_attempts=700,"


~> parmchk2 -s 2 -i ATP.mol2 -f mol2 -o ATP.frcmod


==================================================
Preparing asu files and 4phenix_8icn.pdb
==================================================

~> tleap -f 8icn_asu_tleap_input_run

Checking output filenames
  file : 8icn_asu.prmtop
Sorry:   Output file is empty : 8icn_asu.prmtop
  Check contents of "leap.log"
